src/main/java/com/capitalone/dashboard/service/MetadataServiceImpl.java

```diff
@@ -48,4 +56,12 @@ public String create(MetadataCreateRequest request) throws HygieiaException {
         return entity.getId().toString();
     }
 
+    @Override
+    public DataResponse<Iterable<Metadata>> search(String searchKey, String value) throws HygieiaException {
+         LOGGER.info("MetdataService.search() :  Enter");
+        Iterable<Metadata> mts = customRepositoryQuery.findAllMetaDataBySearchQuery(searchKey,value);
+        return new DataResponse(mts, System.currentTimeMillis());
```

Comment:
Can mts be null?

Reply:
@benj58xu mts will not be null, but it can be empty collection.

Reply:
@benj58xu  logging serach key with value and duration.